src/modules/tracker.js

```diff
@@ -2235,6 +2235,391 @@ class Tracker {
     return new Error('parameters are required of type object');
   }
 
+  /**
+   * Send ASA request submitted event
+   *
+   * @function trackAssistantSubmit
+   * @param {object} parameters - Additional parameters to be sent with request
+   * @param {string} parameters.intent - Intent of of user request
+   * @param {string} [parameters.section] - The section name for the item Ex. "Products"
+   * @param {object} [networkParameters] - Parameters relevant to the network request
+   * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
+   * @returns {(true|Error)}
+   * @description User submitted an assistant search
+   *   (pressing enter within assistant input element, or clicking assistant submit element)
+   * @example
+   * constructorio.tracker.trackAssistantSubmit(
+   *     {
+   *         intent: 'show me a recipe for a cookie',
+   *     },
+   * );
+   */
+  trackAssistantSubmit(parameters, networkParameters = {}) {
+    if (parameters && typeof parameters === 'object' && !Array.isArray(parameters)) {
+      // Ensure parameters are provided (required)
+      const baseUrl = `${this.behavioralV2Url}assistant_submit?`;
+      const {
+        section,
+        intent,
+      } = parameters;
+      const queryParams = {};
+      const bodyParams = {
+        intent,
+        section,
+      };
+
+      if (section) {
+        queryParams.section = section;
+      }
+
+      const requestURL = `${baseUrl}${applyParamsAsString(queryParams, this.options)}`;
+      const requestMethod = 'POST';
+      const requestBody = applyParams(bodyParams, {
+        ...this.options,
+        requestMethod,
+      });
+      this.requests.queue(
+        requestURL,
+        requestMethod,
+        requestBody,
+        networkParameters,
+      );
+      this.requests.send();
+      return true;
+    }
+
+    this.requests.send();
+
+    return new Error('parameters are required of type object');
+  }
+
+  /**
+   * Send assistant results page load started
+   *
+   * @function trackAssistantLoadStarted
```

Comment:
Not a big deal and I don't have a very strong preference but I would prefer `trackAssistantResultLoadStarted`. I think the name is a bit confusing with the word `result` in it

Reply:
I am confused.

You prefer trackAssistantResultLoadStarted. But you also metnion that the name is confusing with the word result in it 😅 .

So are you for or against having result in it 🤔

Reply:
I meant without but I typed with 🤦 Sorry, I think it's a bit confusing when we do not have the word result in it